### PR TITLE
Update UPGRADE-v3.md adding relative links

### PR DESCRIPTION
**Pull request by shakaran**
Add internal relative links to facilitate the direct access for examples in BC breaks.

### CODE REVIEW

File: UPGRADE-v3.md [hunk 1]
```diff
@@ -157,8 +157,8 @@ Code BC changes
 
 - class `Token` is now `final`
 - class `Tokens` is now `final`
-- method `create` of class `Config` has been removed, use the constructor
-- method `create` of class `RuleSet` has been removed, use the constructor
+- method `create` of class `Config` has been removed, [use the constructor](doc/config.rst)
```
Comment (keradus) [inline UPGRADE-v3.md:160]: thanks for PR!
pls, fix the violation detected by CI

Comment (shakaran) [reply UPGRADE-v3.md]: Done

**Review by kubawerlos — APPROVED** [review]

File: UPGRADE-v3.md [hunk 2]
```diff
@@ -157,8 +157,8 @@ Code BC changes
...
+- method `create` of class `RuleSet` has been removed, [use the constructor](doc/custom_rules.rst)
```
Comment (keradus) [inline UPGRADE-v3.md:161]:
```suggestion
- method `create` of class `Config` has been removed, [use the constructor](./doc/config.rst)
- method `create` of class `RuleSet` has been removed, [use the constructor](./doc/custom_rules.rst)
```

ultra minor. somehow we used to prefix local link with `./`. just for consistency

**Comment by keradus** [comment]
Thank you @shakaran.

**Pull request merged**